src/clearCompleted.js

```diff
@@ -0,0 +1,21 @@
+import { createList } from './addtask.js';
+
+export default function clearAll(items) {
+  const checkedTasks = [];
+  const deleteButton = document.querySelector('.delete');
+  deleteButton.addEventListener('click', () => {
+    for (let i = 0; i < items.length; i += 1) {
+      if (items[i].completed === true) {
```

Comment:
You could use the filter function instead, since you wanted to get only the specific "items[I].completed === completed"

It would be something like that:

`items.filter((item) => (item.completed !== true));`